docs/conf.py

```diff
@@ -56,6 +58,25 @@
 html_theme = 'pydata_sphinx_theme'
 # html_static_path = ['_static']
 
+# -- manage thumbnails --------------------------------------------------------
+# define thumbnails
+# (must be in 'resources' and will always be copied to'_images')
+nbsphinx_thumbnails = {
+    'gallery/interactive/pyfar_audio_objects': '_images/thumbnail_pyfar_audio_objects.png',
+}
```

Comment:
```suggestion
nbsphinx_thumbnails = {
    'gallery/interactive/pyfar_audio_objects': '_static/thumbnail_pyfar_audio_objects.png',
}
```
Using the _static folder, the files will automatically get copied into the _build directory. If accepting this, make sure to move the file to the respective folder :)